src/gluonts/model/seq2seq/_transform.py

```diff
@@ -171,7 +171,12 @@ def flatmap_transform(
                         ):
                             dec_field[:] = ts[:, idx : idx + self.dec_len].T
 
-                    out[self._future(ts_field)] = np.squeeze(forking_dec_field)
+                    if forking_dec_field.shape[-1] == 1:
```

Comment:
Shape won’t really work with hybridize=True I guess, or I’m missing something?

Reply:
Sorry, I misunderstood: we're in numpyland

Reply:
Wouldn't it make sense to add a test for this?

Reply:
Yeah we probably should. We should add much more edge case tests in general.